packages/libraries/src/loggingHelper.ts

```diff
@@ -8,15 +47,59 @@ export function getRandomSequenceToken() {
   return token
 }
 
-export async function sendLog(logRoute: string, log: object) {
+const headers = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+}
+
+// helper function for sending frontend logs to the logging route
+export async function addLog(logRoute: string, log: AddLogRequest) {
```

Comment:
Warum muss der Client die LogRoute kennen?

Reply:
Diese Funktionen sind ja lediglich ein Wrapper für den log request (fetch) aus dem frontend in das backend. Meine erste Idee war es diese Funktionen direkt an die Route klasse zu hängen und dann einfach auf die jeweilige path variable zu gucken. Leider kommt man im FE nicht mehr an den path ran. Ausserdem ist es auch nicht schön die Route ins frontend zu importieren. Ich habe hier wirklich viel probiert. Das hier ist aber m.E. die beste Lösung.